tests/sweep_framework/sweeps/eltwise/unary_backward/silu_bw/silu_bw.py

```diff
@@ -28,17 +23,25 @@
         "input_shape": gen_shapes([1, 1, 32, 32], [6, 12, 256, 256], [1, 1, 32, 32], 16)
         + gen_shapes([1, 32, 32], [12, 256, 256], [1, 32, 32], 16)
         + gen_shapes([32, 32], [256, 256], [32, 32], 16),
-        "grad_dtype": [ttnn.bfloat16],
-        "input_a_dtype": [ttnn.bfloat16],
-        "grad_layout": [ttnn.TILE_LAYOUT],
-        "input_a_layout": [ttnn.TILE_LAYOUT],
+        "grad_dtype": [ttnn.bfloat16, ttnn.bfloat8_b],
+        "input_a_dtype": [ttnn.bfloat16, ttnn.bfloat8_b],
```

Comment:
Does silu_bw support bf8? is that updated in the doc string as well?

Reply:
Yes, it supports bfloat8_b, and the documentation has been updated accordingly.